CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.4)
 
 project(inexor-vulkan-renderer)
 
+option(USE_VMA_RECORDING "Use VulkanMemoryAllocator recording feature" On)
```

Comment:
Better use `INEXOR_USE_VMA_RECORDING` its more safe as it prevents name collisions